app/main.py

```diff
@@ -1 +1,27 @@
-# write your code here
+class Animal:
+    alive = []
```

Comment:
The `alive` list is a class variable, which means it is shared across all instances of `Animal`. This can lead to unintended side effects if not managed properly, especially when removing elements. Consider whether this is the intended behavior or if an instance variable would be more appropriate.